### ISSUE TITLE
ReferenceError: __DENO_NODE_GLOBAL_THIS_xxxxxxxxxx__ is not defined

### ISSUE THREAD

**Issue by lambdalisue** [vim-denops/denops.vim#218]
It seems using `npm:diff` cause the following error

![image](https://user-images.githubusercontent.com/546312/206895784-62cbecba-e587-4f3f-a11f-338bfef0636b.png)

### Reproduce

Add the following lines to the bottom of `example-minimal/main.ts` of vim-denops/denops-helloworld.vim

```ts
import * as diff from "npm:diff";
console.log(diff.diffChars("foo", "bar"));
```

Reproduced with

- Deno 1.25.4 with/without `deno.jsonc`
- Deno 1.26.2 with/without `deno.jsonc`
- Deno 1.27.2 without `deno.jsonc`
- Deno 1.28.3 without `deno.jsonc`

Couldn't reproduced with

- Deno 1.27.2 with `deno.jsonc`
- Deno 1.28.3 with `deno.jsonc`

**Comment by lambdalisue** [comment]
We moved to `deno task` thus this issue is no longer exist

**Comment by tani** [comment]
@lambdalisue Excuse me, I have the same issue in the following condition.
Would you mind to confirm this issue again?

# Version Info

Deno version
```
deno 1.32.4 (release, x86_64-unknown-linux-gnu)
v8 11.2.214.9
typescript 5.0.3
```

Neovim version
```
NVIM v0.9.0
Build type: Release
LuaJIT 2.1.0-beta3

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "
/home/linuxbrew/.linuxbrew/Cellar/neovim/0.9.0/share/nvim"

Run :checkhealth for more info
```

Denops version
```
commit d7a15615f86830e9464c30f761a3911f619b38b3 (HEAD -> main, origin/main, origin/HEAD)
Merge: 8f3899d 1363f97
Author: Λlisue (Ali sue・ありすえ) <lambdalisue@gmail.com>
Date:   Mon Apr 24 14:43:04 2023 +0900

    Merge pull request #248 from vim-denops/checkhealth

    Fix for neovim 0.10
```

# Reproducible Steps

I have use `NVIM_APPNAME` to create the sandbox.

1. `$ mkdir -p ~/.config/nvim/nvim-denops/`
2. `$ nvim ~/.config/nvim/nvim-denops/init.vim`
   ```vim
   let &rtp .= ',' .. expand('<sfile>:p:h') .. '/example.vim'
   let &rtp .= ',' .. expand('<sfile>:p:h') .. '/denops.vim'
   ``` 
3. `$ mkdir -p ~/.config/nvim/nvim-denops/example.vim/denops/example`
4. `$ nvim ~/.config/nvim/nvim-denops/example.vim/denops/example/main.ts`
   ```typescript
   import { Denops } from "https://deno.land/x/denops_std@v4.0.0/mod.ts";
   import * as diff from "npm:diff";

   export async function main(denops: Denops) {
     diff
   }
   ```
5. `$ git clone https://github.com/vim-denops/denops.vim ~/.config/nvim-denops/denops.vim`
6. Launch Neovim `nvim`
